src/cinnamon-recorder.c

```diff
@@ -509,20 +509,12 @@ recorder_draw_buffer_meter (CinnamonRecorder *recorder)
                   recorder->horizontal_adjust - 2,  recorder->stage_height - recorder->vertical_adjust - 2);
 }
 
-/* We want to time-stamp each frame based on the actual time it was
- * recorded. We probably should use the pipeline clock rather than
- * gettimeofday(): that would be needed to get sync'ed audio correct.
- * I'm not immediately sure how to handle the adjustment we currently
- * do when pausing recording - is pausing the pipeline enough?
- */
 static GstClockTime
 get_wall_time (void)
 {
-  GTimeVal tv;
-
-  g_get_current_time (&tv);
+  GstClock clock;
 
-  return tv.tv_sec * 1000000000LL + tv.tv_usec * 1000LL;
+  return gst_clock_get_time (&clock);
```

Comment:
It should just fit with the function type so using this function *should* work

Reply:
should be ok